cmd/eval.go

```diff
@@ -316,6 +336,10 @@ func eval(args []string, params evalCommandParams, w io.Writer) (bool, error) {
 		return false, err
 	}
 
+	if params.optimizationLevel > 0 {
+		defer os.Remove(ectx.optimizedBundle) // clean up
```

Comment:
Hmm is it feasible to avoid putting anything into the file system in the first place? 

💭 Perhaps we could use `io.ReadCloser` and `io.WriteCloser` instead in some places, so that we could build the optimized bundle in memory, and consume it from there.

Reply:
Updated the code to avoid creating a temp file and instead keep the optimized bundle in memory.